sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_weekly_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+WITH terminal_queries AS (
+  SELECT
+    array_reverse(ARRAY_AGG(sanitized ORDER BY sequence_no, timestamp ASC))[SAFE_OFFSET(0)].*
+  FROM
+    `moz-fx-data-shared-prod.search_terms_derived.merino_log_sanitized_v3` sanitized
+  WHERE
+    date(timestamp) = @submission_date
```

Comment:
Is the goal to replace the weekly roll up on a daily basis? if so I think what we want here is something like:

```sql
WHERE
  date(timestamp) >= date_trunc(@submission_week, week)
```

Reply:
@quiiver Thanks for the heads up. I was wrong about this -- we still want a daily table, but on each day, we'll look back at the past 7 days and only keep the search terms that have appeared >1 time in the last 7 days. Do you think [this](https://github.com/mozilla/bigquery-etl/blob/5e720044629dbfd27a29b813e8d35c4101bf0951/sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/query.sql#L7) and [this](https://github.com/mozilla/bigquery-etl/blob/5e720044629dbfd27a29b813e8d35c4101bf0951/sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/query.sql#L44) will work?